content/de/docs/rate-limits.md

```diff
@@ -22,7 +22,7 @@ Wenn Sie an der Integration von Let's Encrypt als Provider arbeiten oder
 einer grossen Webseite bitte [lesen Sie unseren Integration Guide]({{< relref "/docs/integration-guide.md" >}}).
 
 Das Hauptlimit ist <a name="certificates-per-registered-domain"></a>**Zertifikate
- pro registrierte Domain**, (50 pro Woche).
+ pro registrierte Domain** (50 pro Woche).
 Eine registrierte Domain, ist generell gesehen, der Teil der Domain, den Sie
```

Comment:
Correct position of the comma
`Eine registrierte Domain ist, generell gesehen,`